app/views/publish/courses/accredited_provider/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= content_for :page_title do %>
+  <%= t(
+    "publish.providers.courses.accredited_provider.show.heading",
+    provider_name: @course.provider_name
```

Comment:
Should this be the Accredited Provider's name here?

Reply:
Good catch